### DISCUSSION TITLE
Support for new Wyze Duo Cam doorbell

### DISCUSSION THREAD

**Discussion by terryhonn** [mrlt8/docker-wyze-bridge#1374] [General]
I haven't searched extensively, but did the specs for the just-announced Duo Cam doorbell get leaked? 

https://www.wyze.com/products/wyze-duo-cam-doorbell

Any ideas if it's using the right architecture for this AMAZING software?

**Comment by spuggy1968** [comment]
Just got one, I hope your great software will be able to integrate this Docker Wyze Bridge!   Thanks

**Comment by thereal-BLKMGK** [comment]
I'm betting Wyze is making it harder, currently some of the floodlights aren't supported and I know they've begun blocking 3rd party software installs with crypto keys. We could probably speed this along by finding a way to get hardware examples to the developer though!

**Comment by dmkjr** [comment]
Since the Wyze Bridge isn't displaying the video from the Doorbell Duo Cam, I wonder if the parameters just need to be updated for the resolution?

**Reply by spuggy1968** [reply]
No, it doesn’t even show up in the log of camera’s, so at this time I don’t think any changes will get it to work.   From what I have seen this device uses different video format “GWell” like the pro cams, and there is no support at this time.
